README.md

```diff
@@ -57,9 +57,9 @@ open FoodTracker.xcodeproj
 ```  
 
 3. Run the project to ensure that it's working  
-    1. Press the Run button or use the ⌘+R key shortcut.  
-    2. Add a meal in the Simulator by clicking the '+' button, providing a name, selecting a photo and a rating and clicking "Save".
-    3. Check that you receive a “Meals successfully saved.” message in the console
+    1. Press the `Run` button or use the `⌘+R` key shortcut.  
+    2. Add a meal in the Simulator by clicking the `+` button, providing a name, selecting a photo and a rating and clicking `Save`.
+    3. Check that you receive a `Meals successfully saved.` message in the console
```

Comment:
Viewing the changes shows that the numbered list is failing to line break (its merging into a single line)